src/common/components/layout/PageContent.tsx

```diff
@@ -5,6 +5,8 @@ import { useMatomo } from '@datapunt/matomo-tracker-react';
 import LoadingContent from '../loading/LoadingContent';
 import styles from './pageContent.module.css';
 
+export const MAIN_CONTENT_ID = 'main-content';
+
```

Comment:
This solution triggers a navigation to `/main-content` and the user sees `404 - page`.